BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
+load("@subpar//:subpar.bzl", "par_binary")
+load("@my_deps//:requirements.bzl", "requirement")
+
+par_binary (
+    name="rss_downloader",
+    srcs=[
+        'rss_downloader.py'
```

Comment:
either use single quotes or double quotes. stay consistent.

also, comma at end of everything: `"rss_downloader.py",`